lib/seed_migration/migrator.rb

```diff
@@ -129,9 +129,8 @@ def self.bootstrap(last_timestamp = nil)
       p "Assume seed data migrated up to #{last_timestamp}"
       files = get_migration_files(last_timestamp.to_s)
       files.each do |file|
-        _, version = parse_migration_filename(file)
         migration = SeedMigration::DataMigration.new
-        migration.version = version
+        migration.version, _ = parse_migration_filename(file)
```

Comment:
I think I get it, but this feels a bit weird since we end up calling `version=` on `migration` and discarding the second result of `parse_migration_filename` at the same time.
Since I assume the test passes I guess that it works as expected, but I'm a little bit confused by it.

It's more that I've never seen that before, the only cases where I've seen multiple returns being used was to assign the result to variables not to call a method on one of the receivers on the left side.

Good to know that it's possible to do.